Objects/unicodeobject.c

```diff
@@ -8964,21 +8964,20 @@ _PyUnicode_InsertThousandsGrouping(
     return count;
 }
 
-
-Py_ssize_t
-PyUnicode_Count(PyObject *str,
-                PyObject *substr,
-                Py_ssize_t start,
-                Py_ssize_t end)
+static Py_ssize_t
+any_unicode_count(PyObject *str,
```

Comment:
I suggest to rename the function to unicode_count_impl(), since most private (static) functions in this C file start with ``unicode_``.